src/envo/envo-edit.owl

```diff
@@ -39106,6 +39194,75 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03510041>
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520003> "cloud part"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520003> <http://purl.obolibrary.org/obo/ENVO_01001784>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520003> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/ENVO_01000760>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520004> (cloud base)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520004> "A cloud part which is closest to the surface of the astronomical body local to the cloud."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520004> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520004> "cloud base"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520004> <http://purl.obolibrary.org/obo/ENVO_03520003>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520005> (cloud top)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520005> "A cloud part which is furthest from the surface of the astronomical body local to the cloud."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520005> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520005> "cloud top"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520005> <http://purl.obolibrary.org/obo/ENVO_03520003>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520006> (supercooled water droplet)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.skybrary.aero/index.php/Supercooled_Water_Droplets>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520006> "A water droplet which exists in liquid form at temperatures below 0°C (32°F)."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03520006> <http://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03520006> "supercooled water droplet"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03520006> <http://purl.obolibrary.org/obo/ENVO_01000833>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03520007> (snow crystal)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.snowcrystals.com/science/science.html>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03520007> "A water ice crystal within which the water molecules are all lined up in a precise hexagonal array."@en)
```

Comment:
The definition could be refined a little and the source seems a little shaky - please add one more corroborating source. 

Try: "A water ice crystal which is constituted by water molecules aligned in a hexagonal array."

Also, please add a rdfs:comment to your broad synonym explaining this a little bit - it's quite interesting but tricky because the term is so widely used.

Reply:
You could also think about linking this class to [snow ](http://purl.obolibrary.org/obo/ENVO_01000406) via a subclass axiom

Reply:
Thanks! I've adopted the definition you suggested and added a rtfs:comment to the broad synonym.

> You could also think about linking this class to snow via a subclass axiom

As of now, snow is said to be `'composed primarily of' some 'water ice'`, `'output of' some snowfall`, `'has part' some air`, and `environmental material`. I think the definition of [snow](http://purl.obolibrary.org/obo/ENVO_01000406) would allow for another axiom like this to further specify the 'water ice':
`'composed primarily of' some ('object aggregate' and ('has member' some 'snow crystal'))`. This could be extended to include _snowflake_ when _snowflake_ is added to ENVO